tools/setup_emscripten_win.bat

```diff
@@ -1,11 +1,13 @@
-pushd %~dp0\..
-
-call git clone https://github.com/emscripten-core/emsdk.git
-call cd emsdk
-call emsdk install 3.1.35
-call emsdk activate 3.1.35
-call emsdk install mingw-7.1.0-64bit
-call emsdk activate mingw-7.1.0-64bit
-call cd ..
-
-popd
+pushd %~dp0\..
+
+call git clone https://github.com/emscripten-core/emsdk.git
+call cd emsdk
+::call emsdk install 3.1.35

```

Comment:
It is intentionally tied to a fixed version so every developer has the exact same environment. If you would like to update, please use the latest fixed version.